dsl/scripts/jobs.groovy

```diff
@@ -280,6 +281,8 @@ def lnlSystemTestsparams() {
         stringParam('TEST_EXTRA_PYTEST_ARGS', '', 'extra args passed to system tests executiom')
         stringParam('CAPSULE_CONFIG', 'capsule_config_mainnet.hcl', 'Run tests using the given vegacapsule config file')
         booleanParam('SKIP_MULTISIGN_SETUP', true, h('When true validators are not added to multisig as signers'))
+        booleanParam('BUILD_PROTOCOL_UPGRADE_VERSION', true, 'If true, temporary release is created under the vegaprotocol/vega-dev-releases. Release is used for protocol upgrade tests. There are two environment variables available for system-tests to find that release: `PROTOCOL_UPGRADE_EXTERNAL_RELEASE_REPOSITORY`, `PROTOCOL_UPGRADE_EXTERNAL_RELEASE_VERSION`')
```

Comment:
You can't define same param twice (you already have it in BUILD_PROTOCOL_UGPRADE_VERSION), in particular: it won't get overwritten. You need to define it separately for each place that requires it if default values should be different.

You can do it by adding argument to systemTestsParamsGeneric function